docs/tutorial.rst

```diff
@@ -3,6 +3,113 @@
+models in the ``stdpopsim`` :ref:`catalog <sec_catalog>`. The other way to
+access the ``stdpopsim`` resources is via the python API. This is a bit more
+complicated however it allows for more advanced tasks. This tutorial will walk
+through both ways of using the ``stdpopsim`` package as well as some more
+advanced tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+In order to use the ``stdpopsim`` CLI the ``stdpopsim`` package must be
+installed (see :ref:`Installation <sec_installation>`). The CLI provides access
+to the :ref:`catalog <sec_catalog>` of models that have already been implemented
+by ``stdpopsim``. The first step for using the CLI is to select the species that
+you are interested in simulating data for. In order to see which species are
+available run
+
+.. code-block:: console
+
+    $ stdpopsim --help
+
+This shows the species currently supported by ``stdpopsim``. This means that
+``stdpopsim`` knows various traits of these species including chromosome size
+and recombination rates. Once we've selected a species, in this case humans, we
+can look at the help again as follows.
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help
+
+For conciseness we do not show the output here but this time you should see a
+different output which shows options for performing the simulation itself and
+the species default parameters. This includes selecting the demographic model,
+chromosome, recombination map, and number of samples. 
+
+The most basic simulation we can run is to draw two samples using the species'
+defaults as seen in the species help (``stdpopsim homsap --help``). These
+defaults include constant size population, uniform recombination map, and a
+default chromosome. To save time we will also specify that the simulation use
+chromosome 22 with the ``-c`` option. 
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o foo.ts 2
+
+.. warning:: It's important to remember to either redirect the output of ``stdpopsim``
+                to file or to use the ``-o/--output`` option. If you do not, the 
+                binary output may mess up your terminal session.
+
+Say we want to use a specific demographic model. We look up the available models
+using the ``--help-models`` flag:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This gives all of the possible demographic models we could simulate. We choose
+the the two population out-of-Africa model from `Tennesen et al.
+<https://doi.org/10.1126/ science.1219240>`_ . By looking at the model help we
+find that the name for this model is ``ooa_2`` and that we can specify it using
+the ``--model`` option. We choose to draw two samples from the African
+population and three samples from the European population. To increase
+simulation speed we can also chose to simulate only a subset of the chromosome
+using the ``-l`` option (e.g. 5%). The command now looks like this:
```

Comment:
See my new issue @216 -- I think this 'length-multiplier' option isn't well described yet. Which subset of the chromosome are you simulating -- are you just simulating the first 5%? Or are you effectively simulating a pruned subset of SNPs in the chromosome so you have 5% of them? I think you should change this sentence to something more explicit like this.

Reply:
yes I agree it's vague. we should be sure to point out to folks that you can't use length-multiplier option with anything other than a uniform recombination map.